site/components/pages/Pledge/lib/formSchema.ts

```diff
@@ -1,11 +1,29 @@
 import { t } from "@lingui/macro";
+import { ethers } from "ethers";
```

Comment:
I'm not sure how well tree shaking works for ethers but this is probably better:
```suggestion
import { utils } from "ethers";
```

Reply:
Classic copy pasta stitch up.
Thanks for fixing that one up @goatsOnDrugs!